javascript/javascript1/week1/week-1.js

```diff
@@ -0,0 +1,73 @@
+//* Age-ify (A future age calculator) *//
+
+const yearOfBirth = 2005;
+const yearFuture = 2076;
+
+const age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture);
+
+//* Goodboy-Oldboy (A dog age calculator) *//
```

Comment:
Good idea to add organizational comments into your code! Makes it easy to read! ⚡